Homework/Duy_Truong/922_sort_by_parity_2.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {number[]} nums
+ * @return {number[]}
+ */
+var sortArrayByParityII = function(nums) {
+    let evenArr = []; let oddArr = []; let res = [];
```

Comment:
can you improve this to O(1) extra space? hint: Dutch flag partition

Reply:
I've resolve it